botorch/models/gp_regression.py

```diff
@@ -159,13 +179,26 @@ def forward(self, x: Tensor) -> MultivariateNormal:
 class FixedNoiseGP(BatchedMultiOutputGPyTorchModel, ExactGP):
     r"""A single-task exact GP model using fixed noise levels.
 
-    A single-task exact GP that uses fixed observation noise levels. This model
-    also uses relatively strong priors on the Kernel hyperparameters, which work
+    A single-task exact GP that uses fixed observation noise levels, differing from
+    `SingleTaskGP` only in that noise levels are provided rather than inferred.
+    This model also uses relatively strong priors on the Kernel
+    hyperparameters, which work
     best when covariates are normalized to the unit cube and outcomes are
     standardized (zero mean, unit variance).
```

Comment:
```suggestion
    This model also uses relatively strong priors on the Kernel hyperparameters,
    which work best when covariates are normalized to the unit cube and outcomes
    are standardized (zero mean, unit variance).
```

Reply:
This is just about the placement of line breaks? FWIW this doesn't affect how it renders (although the suggested change does look better in code).

I wish there was a way to automatically fix docstring line length since this was tedious, but it looks like the tools that exist for this (`docstringformatter` and `pydocstringformatter`) make too many other changes at the same time.